internal/pkg/cli/storage_init.go

```diff
@@ -532,10 +532,10 @@ func (o *initStorageOpts) RecommendedActions() []string {
 
 	newVar := template.ToSnakeCaseFunc(template.EnvVarNameFunc(o.storageName))
 
-	svcDeployCmd := fmt.Sprintf("copilot svc deploy --name %s", o.storageSvc)
+	svcDeployCmd := fmt.Sprintf("copilot svc deploy --name %s", o.storageWl)
 
 	return []string{
-		fmt.Sprintf("Update your service code to leverage the injected environment variable %s", color.HighlightCode(newVar)),
+		fmt.Sprintf("Update your service or job code to leverage the injected environment variable %s", color.HighlightCode(newVar)),
 		fmt.Sprintf("Run %s to deploy your storage resources to your environments.", color.HighlightCode(svcDeployCmd)),
```

Comment:
Can we update this so that it's just "copilot deploy" instead of "copilot svc deploy"? and maybe ~~to your environments.~~